### ISSUE TITLE
Request: Injection Mode

### ISSUE THREAD

**Issue by PureOcean** [FastForwardTeam/FastForward#1139]
### Body

First of all, thanks to everyone who made efforts for MV3. I am pleased to have added new options page and white list feature.

[Code Verify](https://addons.mozilla.org/firefox/addon/code-verify/) add-on incompatibility was [mentioned](https://github.com/FastForwardTeam/FastForward/issues/482) in last year.

Until now, the only solution was to disable FastForward. Most of the time I couldn't use FF because I forgot to activate it again. I am glad that the white list feature with MV3. Would it be possible to actively use FF from now on?

I added `https://web.whatsapp.com/*` to the WhiteList section in new settings page .Unfortunately it didn't work. Code Verify extension still breaking when FlashForward is enabled.

This problem also occurs with Tampermonkey. Thanks to a [special setting](https://github.com/Tampermonkey/tampermonkey/issues/1846#issuecomment-1706640339) proposed by the developer, Code Verify has now started to work properly with Tampermonkey.

This feature is [instant inject mode](https://stackoverflow.com/a/69928641). Is it possible to add this feature to FF as a experimental option? I think this is the only way for the Whitelist feature to work in compatiable with Code Verify extension.

### Version

Manifest-v3

### What browser(s) are you seeing the problem on?

Firefox browsers

### What platform are you seeing the problem on?

Computer (Windows, Linux, MacOS)

**Comment by lostdusty** [comment]
Maybe this could help us fix the consent dialog bug present in Firefox. I'll assign not-a-proton to this.

**Comment by NotAProton** [comment]
FastForward injects scripts because your whitelist format is incorrect, it clearly says to add sites as domain.tld. Add `web.whatsapp.com` to your whitelist and FastForward won't inject anything.
![image](https://github.com/FastForwardTeam/FastForward/assets/81463636/feae82db-eb3e-42e4-b22f-61a87ec282b5)

**Comment by PureOcean** [comment]
@princessmortix and @NotAProton, Wow. Thank you so much.

the problem that did not exist was solved. When Fast Forward is now enabled, Code Verify shows green light in WhatsApp Web page.

Without writing this report, I should read FF's options page correctly and add only `web.whatsapp.com` to Whitelist as mentioned there. I thoughtlessly used the whitelist syntax string of other extensions in FF. I'm sorry, I took your time.

On the other hand, I'm happy to use Fastforward again. And I am grateful for your continuously efforts in this project.
